r/R/rzeppelin/R/rServer.R

```diff
@@ -0,0 +1,214 @@
+rServe <- function(sockets) {
```

Comment:
Could you explain please, this looks different from https://github.com/cran/rscala/blob/master/R/rServer.R - or am I looking at the wrong place?